packages/google_maps_flutter/google_maps_flutter_ios/ios/Classes/GoogleMapMarkerController.m

```diff
@@ -168,131 +139,91 @@ - (void)interpretInfoWindow:(NSDictionary *)data {
   }
 }
 
-- (UIImage *)extractIconFromData:(NSArray *)iconData
-                       registrar:(NSObject<FlutterPluginRegistrar> *)registrar
-                     screenScale:(CGFloat)screenScale {
+- (UIImage *)iconFromBitmap:(FGMPlatformBitmap *)platformBitmap
+                  registrar:(NSObject<FlutterPluginRegistrar> *)registrar
+                screenScale:(CGFloat)screenScale {
   NSAssert(screenScale > 0, @"Screen scale must be greater than 0");
+  // See comment in messages.dart for why this is so loosely typed.
```

Comment:
Maybe add the issue link for searchability? https://github.com/flutter/flutter/issues/117819

Reply:
Done.